docker-app/qfieldcloud/urls.py

```diff
@@ -42,14 +42,19 @@
         name="index",
     ),
     path(
-        "schema/",
+        "swagger.yaml",
```

Comment:
I think it's not considered a good practice to use file extensions in urls, for the sake of not being hostage of an extension. But I am no expert in these conventions.

Reply:
Well, it depends. For the sake of backwards compatibility, it should have the extension for now.